tests/test_content_smart_proxy.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+
+@pytest.fixture
+def content_smart_proxy(api, organization):
+    smart_proxies = api.resource('smart_proxies').call('index')
+    content_proxy = {}
+
+    for proxy in smart_proxies['results']:
+        for feature in proxy['features']:
+            if feature['name'] == 'Pulp Node':
+                content_proxy = proxy
+                break
+
+    content_proxy_id = content_proxy['id']
```

Comment:
If the proxy is not found, this raises a KeyError. You will hit this guaranteed on a Pulp 3 only setup where you need to search for a Pulpcore feature with the mirror setting set to true. I don't think settings are exposed (the are considered private) so we should figure out a different way to detect this. Perhaps it should be a capability (which I think is exposed).